Description: Remove the bits of CMakeLists.txt we don't need
Author: Dave Hibberd <d@vehibberd.com>
Last-Update: 2018-02-20

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1625,8 +1625,8 @@ if (UNIX)
     add_dependencies (wsjtx manpages)
   endif (NOT WSJT_SKIP_MANPAGES)
   if (NOT APPLE)
-    add_subdirectory (debian)
-    add_dependencies (wsjtx debian)
+    #add_subdirectory (debian)
+    #add_dependencies (wsjtx debian)
   endif (NOT APPLE)
 endif (UNIX)
 
@@ -1673,27 +1673,6 @@ install (TARGETS ft8code jt65code jt9cod
   )
 endif(WSJT_BUILD_UTILS)  
 
-install (PROGRAMS
-  ${RIGCTL_EXE}
-  DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #COMPONENT runtime
-  RENAME rigctl-wsjtx${CMAKE_EXECUTABLE_SUFFIX}
-  )
-
-install (PROGRAMS
-  ${RIGCTLD_EXE}
-  DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #COMPONENT runtime
-  RENAME rigctld-wsjtx${CMAKE_EXECUTABLE_SUFFIX}
-  )
-
-install (PROGRAMS
-  ${RIGCTLCOM_EXE}
-  DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #COMPONENT runtime
-  RENAME rigctlcom-wsjtx${CMAKE_EXECUTABLE_SUFFIX}
-  )
-
 install (FILES
   README
   COPYING
--- a/wsjtx.pro
+++ b/wsjtx.pro
@@ -14,6 +14,7 @@ VERSION = "Not for Release"
 TEMPLATE = app
 DEFINES = QT5
 QMAKE_CXXFLAGS += -std=c++11
+QMAKE_CXXFLAGS += -std=c++11 $$system(pkg-config --cflags-only-I Qt5SerialPort)
 DEFINES += PROJECT_MANUAL="'\"http://www.physics.princeton.edu/pulsar/K1JT/wsjtx-doc/wsjtx-main.html\"'"
 
 isEmpty (DESTDIR) {
